package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cypress-install2",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
@AmnaShahid96 delete this script and write your own script to run your tests written above.